ui/narrative/methods/view_expression_interactive_heatmap/display.yaml

```diff
@@ -34,16 +34,15 @@ suggestions :
 parameters :
     param0 :
         ui-name : |
-            Expression matrix
+            ExpressionMatrix
         short-hint : |
-           Expression matrix 
-        long-hint  : |
-           Expression matrix 
+           Select the ExpressionMatrix to visualize.
 
 
 description : |
-    View heatmap 
+  <p><p>This is a visualization App to view and explore an ExpressionMatrix as a heatmap. Features are displayed as rows, and the conditions as columns. Mouse over the heatmap cells to view the expression value for a given feature and a condition.</p>
```

Comment:
Got a double-<p> here. Should only need one, I think. The extra one might add some extraneous padding?

Reply:
Oops! That was just a copy/paste error on my part. Sorry!

Reply:
I can fix that real quick